go.mod

```diff
@@ -36,6 +36,8 @@ require (
 	github.com/cockroachdb/logtags v0.0.0-20230118201751-21c54148d20b // indirect
 	github.com/cockroachdb/pebble v0.0.0-20230209160836-829675f94811 // indirect
 	github.com/cockroachdb/redact v1.1.3 // indirect
+	github.com/consensys/bavard v0.1.13 // indirect
+	github.com/consensys/gnark-crypto v0.9.1-0.20230105202408-1a7a29904a7c // indirect
```

Comment:
It appears that the relay has the same problem as the client; mixed indirect dependency versions is causing a newer-than-expected version of `gnark-crypto` to be used (`v0.9.1` vs `v0.7.1`). A solution is to use `replace` in the `go.mod` file [like the client does](https://github.com/flashbots/mev-boost/blob/ce70840d07af5e7e3bf727e223921646b26bbcd3/go.mod#L84-L90).

* https://github.com/flashbots/mev-boost/pull/479#discussion_r1142216718

Reply:
great point, added